### ISSUE TITLE
Dashboard Widget Creation Flow

### ISSUE THREAD

**Issue by ale-cota** [getsentry/sentry#47005]
**Problem**: When creating dashboard widgets based on Performance metrics, we only support a [limited list of filters](https://docs.sentry.io/product/alerts/create-alerts/metric-alert-config/#tags--properties).

**Solution**: Notify the user when they're trying to use a tag/property unsupported by metrics 

**Acceptance Criteria:**
- Re-introduce the alert under the graph **with new copy**: “Based on your search criteria, the sampled events available may be limited and may not be representative of all events.”

![](https://user-images.githubusercontent.com/26696515/230350422-fb26bc70-e4a6-4822-8014-c2ec0e8d5052.png)

**Comment by ale-cota** [comment]
Need to update the docs in widget creation to let people know that the filters are limited

**Comment by ale-cota** [comment]
@matejminar I'm thinking we should do the same thing we do with Alerts also with Dashboards. What's the value of a dashboard that based on the filtering condition is sometimes total events and sometimes indexed events? 🙈 I just wouldn't let people create them anymore if the filter isn't supported by Metrics. And we only let them do it in Discover where the whole point is to find sample events. What do you think?
Cc: @jan-auer, @danielkhan

**Comment by wmak** [comment]
@ale-cota 
I disagree with that, a big workflow we've developed from discover to dashboards is creating dashboard widgets from Discover queries
![image](https://user-images.githubusercontent.com/4205004/231493244-7a6413bf-a668-408c-8356-fdc6525c1149.png)

And vice versa going from a dashboard to a discover query
![image](https://user-images.githubusercontent.com/4205004/231493478-9192680c-04b6-41a5-84a1-c1db0d21fd53.png)

As well, what happens to existing queries that are already based on events?

**Comment by ale-cota** [comment]
Thanks, @wmak! We also had a discussion in the meanwhile and indeed, it makes sense to keep it as is.
